bridges/snowbridge/parachain/runtime/kusama-common/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+//! # Kusama Common
+//!
+//! Config used for the Kusama asset hub and bridge hub runtimes.
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::parameter_types;
+use xcm::opaque::lts::NetworkId;
+
+pub const INBOUND_QUEUE_MESSAGES_PALLET_INDEX: u8 = 80;
```

Comment:
one comment please

Reply:
@claravanstaden Can we also shorten this constant name to `INBOUND_QUEUE_PALLET_INDEX`? The `MESSAGES` part is unnecessary